include/open62541/plugin/nodestore.h

```diff
@@ -411,6 +412,11 @@ struct UA_NodeHead {
     UA_MonitoredItem *monitoredItems; /* MonitoredItems for Events and immediate
                                        * DataChanges (no sampling interval). */
 #endif
+
+    size_t rolePermissionsSize;
+    UA_RolePermissionType *rolePermissions;
+    size_t userRolePermissionsSize;
```

Comment:
The user-role-permissions are the result of compating the userroles and the role-permissions.
They are different for every user.
So it doesn't make sense to store them in the node for all users at once.